src/pocketmine/entity/Effect.php

```diff
@@ -223,8 +223,16 @@ public function setVisible($bool){
 	}
 
 	/**
+	* Returns the level of this effect.
```

Comment:
Would be best to add documentation here to explain the difference between "level of this effect" and "amplifier".